src/ert/_c_wrappers/enkf/config/gen_kw_config.py

```diff
@@ -155,13 +157,13 @@ def get_priors(self) -> List["PriorDict"]:
             )
         return priors
 
-    def transform(self, index, value):
+    def transform(self, index: int, value: float) -> float:
         return self._transform(index, value)
 
     @staticmethod
     def values_from_files(
         realizations: List[int], name_format: str, keys: List[str]
-    ) -> npt.ArrayLike:
+    ) -> npt.NDArray:
```

Comment:
I might be wrong but I think you need to add the `dtype`, ie `npt.NDArray[np.float64]`.